.github/scripts/container-utils.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
```

Comment:
```suggestion
#! /usr/bin/env bash
```

For maximum compatibility (e.g. NixOS)